mod/metrics/spec/lib/formula/javascript/script_metrics.test.coffee

```diff
@@ -0,0 +1,14 @@
+#require('../../../../lib/javascript/script_metrics.js.coffee')
+
```

Comment:
I fixed the issue why this test file didn't run in this PR #871.
(the error about "pairs-editor" : https://semaphoreci.com/ethan/wikirate/branches/pull-request-868/builds/21) 

But I don't think it's possible to access the methods that Ethan wants to be tested in a test file (like `valuesAreValid`). As far as I know you can call them only in the file where they are defined.
I guess we have to add them to `wikirate`, `decko` or `$` if we want to test them but it doesn't seem to be good practice to me to move these helper methods, that we use nowhere else, to those objects.

Reply:
We could solve the problem with exporting the methods we want to test.
e.g. script_metric.js.coffee:
```coffee
exports.DIGITS_AFTER_DECIMAL = 2
exports.valuesAreValid = (hash, multiplier) ->
   ...
```
and then in script_metrics.test.coffee
`$.extend global, require('../../../../lib/javascript/script_metrics.js.coffee')`
```